test/framework/time.ts

```diff
@@ -27,6 +30,28 @@ export async function occurrenceAtMost(
     }
 }
 
+export async function getTimestampNow(): Promise<number> {
+    return (await provider.getBlock('latest')).timestamp
+}
+
+export function advanceBlock() {
+    return provider.send('evm_mine', [])
+}
+
+export const increaseTime = async (time: number, advance = true) => {
```

Comment:
question: why both use an exported function for `setTime` and an exported const for `increaseTime`.

Why not an exported function for both, what the motivation for the inconsistency?